internal/pkg/exec/docker.go

```diff
@@ -230,6 +246,21 @@ func (c DockerCommand) IsEcrCredentialHelperEnabled(uri string) bool {
 	return false
 }
 
+// ValidatePlatform checks if the entered string is a Docker-buildable platform.
+func ValidatePlatform(platform string) error {
+	if platform == "" {
+		return nil
+	}
+	osArch := strings.Split(platform, "/")
+	if len(osArch) < 2 {
+		return fmt.Errorf("platform %s is invalid; must be of format 'os/arch'", platform)
+	}
+	if osArch[0] != LinuxOS || osArch[1] != Amd64Arch {
```

Comment:
Great question! I had written this validation before adding the `validPlatforms` var, so breaking them up seemed to avoid a laundry list of combinations in the future, but you're right-- we should just validate the platform as a whole. Changing!